mne_bids/mne_bids.py

```diff
@@ -538,7 +575,15 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         A path to the configuration file to use if the data is from a BTi
         system.
     overwrite : bool
-        If the file already exists, whether to overwrite it.
+        Whether to overwrite existing files or data in files.
+        Defaults to False.
+        If overwrite is True, any existing files with the same BIDS parameters
+        will be overwritten with the exception of the `participants.tsv` and
+        `scans.tsv` files. These will have only matching pre-existing data
+        replaced.
+        If overwrite is False, an `OSError` will be raised if the file exists
+        already or if there is matching data already in the `participants.tsv`
+        or `scans.tsv` files.
```

Comment:
again, I don't think we need `L584-586` in the docstring. It's too much information for user to digest.

Reply:
actually no, you need these lines but you need to rephrase it. Talk in terms of data being added not in terms of `OSError` ...

Reply:
Perhaps rephrase "These will have only matching pre-existing data replaced." to --> "For these files, parts of pre-existing data that perfectly match the current data will be replaced"

and then

"If overwrite is False, no existing data will be overwritten or replaced"

Reply:
thanks @sappelhoff for those. I was struggling to come up with eloquent ways of describing it. I dropped the `perfectly` as it will overwrite the data with the same filename/subject_id even if the other data is different.